src/main/java/edu/wpi/first/pathui/FxUtils.java

```diff
@@ -22,4 +25,37 @@ public static MenuItem menuItem(String text, EventHandler<ActionEvent> eventHand
     return menuItem;
   }
 
+  /**
+   * Get the color according to sub-child order.
```

Comment:
When writing single sentence code comments insert a hidden "The code" to the front of the sentence.  This helps make the grammar correct.

So here it would be:
The code "gets the color according to sub-child order".

TLDR: `Gets the color according to sub-child order.`